analyses/tmb-compare/compare-tmb-calculations.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Tumor Mutation Burden Comparison of Calculations"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL
+date: 2019
+---
+
+#### Objectives
+
+Compare D3b and CCDL TMB calculations as a response to this [GitHub comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/729#issuecomment-658259843). 
+
+This includes CCDL TMB calculations that used mutation sets with FoCR-defined nonsynonymous filter and maftools-defined nonsynonymous filter. 
+
+We generally treat CCDL's maftools-defined filtered dataset as the benchmark since its the TMB currently in the data release (as of 09-09-2020). 
+
+These TMB datasets compared here are obtained by running:
+
+```
+bash analyses/snv-callers/explore_variant_classifications/run_explorations.sh
+```
+
+### Summary of findings: 
+
+Mutation counts seem to be fairly related but the bed sizes are very different and that seems to be what's causing the differences in TMB calculations ranges.
+CCDL TMB is a much larger range (0 - ~300), while D3B TMB is smaller range (0 - ~5) because the BED size for CCDL TMB is much smaller than the BED size for D3B TMB. 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+    - [Usage](#usage)
```

Comment:
Markdown doesn't seem to like starting with an interior node here, and treats this as a code block. I think changing to `### Usage` may be enough to fix this. *You probably also want to promote `Packages and functions` to H3.